client.go

```diff
@@ -62,16 +47,6 @@ type IRITAClient struct {
 	Oracle   oracle.Client
 }
 
-// AppCodec return a Marshaler of the protobuf
-func AppCodec(rs ...codec.RegisterInterfaces) codec.Marshaler {
-	encodingConfig := makeEncodingConfig()
-	registers = append(registers, rs...)
-	for _, register := range registers {
-		register(encodingConfig.InterfaceRegistry)
-	}
-	return encodingConfig.Marshaler
-}
-
```

Comment:
This method provides that you can use the codec function without initializing the client